samples/java_layout/src/main/res/layout/fragment_attributes_tracking.xml

```diff
@@ -1,83 +1,92 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
These layout files are the same, except I added a `NestedScrollView` as top level child for scrolling. The indentation change caused most of the diff, but there are no other updates in these files.